HW06-Annotation/src/main/java/com/akproductions/hw06_main.java

```diff
@@ -0,0 +1,124 @@
+package com.akproductions;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+public class hw06_main {
```

Comment:
```suggestion
public class HW06Main {
```

Принято классы именовать с большой буквы.